### PR TITLE
Migration/add field video

### PR DESCRIPTION
**Pull request by corneliusgdahling**
Related to OMEGA-53
- Added video asset to embeddedVideo
- renamed (not id) to Video
- made Adresse optional
- Created video field which links to embeddedVideo content type
- Added Video field to article, category and mainCategory

### CODE REVIEW

File: migrations/2020-08-28-155042-add-video-field.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  editField(
+    mergeDeepRight(fieldConfigs.url.v1, {
+      options: mergeDeepRight(fieldConfigs.url.v1.options, { required: false }),
+    }),
+    embeddedVideo
+  )
+
+  const article = migration.editContentType(ContentTypes.Article)
+  const category = migration.editContentType(ContentTypes.CategoryPage)
+  const mainCategory = migration.editContentType(ContentTypes.MainCategory)
```
Comment (JonatanPaalsson) [inline migrations/2020-08-28-155042-add-video-field.ts:38]: Forside?

**Review by JonatanPaalsson — CHANGES_REQUESTED** [review]

File: migrations/2020-08-28-155042-add-video-field.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import Migration from 'contentful-migration'
+import { createField, fieldConfigs, editField } from './fields'
+import { ContentTypes } from './content-types/types'
+import { FieldIds, FieldNames } from './fields/types'
+import { mergeDeepRight } from 'ramda'
+
+module.exports = function runMigration(migration: Migration) {
+  const embeddedVideo = migration.editContentType(ContentTypes.EmbeddedVideo)
```
Comment (t0ff3r) [inline migrations/2020-08-28-155042-add-video-field.ts:8]: rename ContentType to Video and not EmbeddedVideo?

**Review by t0ff3r — CHANGES_REQUESTED** [review]

File: migrations/2020-08-28-155042-add-video-field.ts (hunk 1 above)
Comment (corneliusgdahling) [reply migrations/2020-08-28-155042-add-video-field.ts:38]: done

File: migrations/2020-08-28-155042-add-video-field.ts (hunk 2 above)
Comment (corneliusgdahling) [reply migrations/2020-08-28-155042-add-video-field.ts]: Fixed

**Review by JonatanPaalsson — APPROVED** [review]

**Pull request merged**